metpy/calc/thermo.py

```diff
@@ -840,3 +842,180 @@ def relative_humidity_from_specific_humidity(specific_humidity, temperature, pre
     return (100 * units.percent *
             mixing_ratio_from_specific_humidity(specific_humidity) /
             saturation_mixing_ratio(pressure, temperature))
+
+
+@exporter.export
+@check_units('[temperature]', '[pressure]', '[temperature]')
+def get_isentropic_pressure(isentlevs, lev, tmp, *args, **kwargs):
```

Comment:
How about `theta_levels`,  `pressure`, and `temperature` for the argument names? (You'll need to update the docs as well.)

How does the name `isentropic_interpolation` sound? `interpolate_isentropic_surfaces`? I'm open to other suggestions, but I don't think the current name conveys what's going on.